### PR TITLE
Homepage redesign recently requested

### PR DESCRIPTION
**Pull request by wrightmartin**
This reorganises and redesigns the 'recently requested' section of the homepage, part of the bigger https://github.com/mysociety/alaveteli/issues/1525 ticket

screen shot 2015-05-20 at 17 05 15

### CODE REVIEW

**Comment by garethrees** [comment]
Looks great but might be worth making the body and request titles more obviously clickable?

**Comment by wrightmartin** [comment]
Ready for review again - tweaked the inks a bit

screen shot 2015-06-05 at 12 28 54

File: assets/stylesheets/responsive/custom.scss [hunk 1]
```diff
@@ -827,6 +827,55 @@ p.ui-dropdown-list-trigger {
...
+.icon-standalone {
+  display: block;
+  width: 22px;
+  height: 22px;
+}
+
+.latest-requests {
+  h4 {
+    font-size: 1.125em;
+    color: #9797A4;
```
Comment (garethrees) [inline assets/stylesheets/responsive/custom.scss]: Should apply the greyer font style to the left hand column too:

screen shot 2015-06-05 at 12 50 35

**Comment by garethrees** [comment]
Looks great. Minor comments then we'll be ready to ship it.

**Pull request merged**